tika/server.go

```diff
@@ -145,12 +145,21 @@ const (
 	Version114 version = "1.14"
 	Version115 version = "1.15"
 	Version116 version = "1.16"
+	Version117 version = "1.17"
+	Version118 version = "1.18"
+	Version119 version = "1.19"
 )
 
 var md5s = map[version]string{
```

Comment:
I think we can drop these older versions. This is only used when downloading a new server, in which case it's better to use a newer version. That should simplify a bit of the logic (similar to before with only a single algorithm).

Reply:
What version will you wish to support in server.go? only latest(v1.19)? then shall I remove all old versions?

Reply:
Only the latest sounds good to me.